gossip/poset_hook.go

```diff
@@ -76,7 +76,7 @@ func (hook *HookedEngine) LastBlock() (idx.Block, hash.Event) {
 // GetValidators returns validators of current epoch.
 func (hook *HookedEngine) GetValidators() *pos.Validators {
 	if hook.engine == nil {
-		return pos.NewValidators()
+		return pos.NewBuilder().Build()
```

Comment:
line 79 is dup of line 63. Can make the validators once and cache it for whole epoch?

Reply:
pos.NewBuilder().Build() means "build empty validators". refactored

Reply:
pos.NewValidators() looks cleaner